Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'ra10ke'
 gem 'rubocop'
 gem 'rubocop-rspec'
 gem 'rest-client'
+gem 'onceover'
```

Comment:
Can you pull this out and make adding Onceover a separate PR? Just trying to make it easier to review and approve each set of changes. A little hard to figure out why tests are failing, and having smaller PRs would help with that.

Reply:
Oh, and tests are failing right now. Any idea why? Output looks kinda ambiguous. Complaining about a fixtures file... sounds more related to updates to the Rakefile and test definitions maybe than the actual Puppet code itself.